KerbalStuff/blueprints/api.py

```diff
@@ -184,18 +184,17 @@ def _update_image(old_path: str, base_name: str, base_path: str) -> Optional[str
 
 def _get_modversion_paths(mod_name: str, friendly_version: str) -> Tuple[str, str]:
     mod_name_sec = secure_filename(mod_name)
-    storage_base = os.path.join(f'{secure_filename(current_user.username)}_{current_user.id!s}',
-                                mod_name_sec)
+    base_path = os.path.join(f'{current_user.base_path()}', mod_name_sec)
```

Comment:
```suggestion
    base_path = os.path.join(current_user.base_path(), mod_name_sec)
```

Do we need the `f'{}'` here? Looks like a single value returned with no extra formatting.

Reply:
Good spot, a left-over from before.